src/script/conversation/ConversationVerificationStateHandler/MLS/MLSStateHandler.test.ts

```diff
@@ -26,10 +26,27 @@ import {Core} from 'src/script/service/CoreSingleton';
 import {createUuid} from 'Util/uuid';
 import {waitFor} from 'Util/waitFor';
 
-import {registerMLSConversationVerificationStateHandler} from './MLSStateHandler';
+import {MLSConversationVerificationStateHandler} from './MLSStateHandler';
+
+const createMLSConversationVerificationStateHandler = (
+  domain: string,
+  onConversationVerificationStateChange: OnConversationE2EIVerificationStateChange,
+  onSelfClientCertificateRevoked: () => Promise<void>,
+  conversationState: ConversationState,
+  core: Core,
+) => {
+  return new MLSConversationVerificationStateHandler(
+    domain,
+    onConversationVerificationStateChange,
+    onSelfClientCertificateRevoked,
+    conversationState,
+    core,
+  );
+};
```

Comment:
I guess we could just remove this abstraction layer and just expose the class itself, I don't see the value of having the factory function that just wraps the constructor

Reply:
You're right, done with https://github.com/wireapp/wire-webapp/pull/17056/commits/e36efcce84b0cb7504572becf14bfab4e9df701c